packages/fast-components-react-base/src/divider/docs.md

```diff
@@ -0,0 +1,11 @@
+## Best practices
+Use the *divider* to show a change in the content between two blocks of information. The content surrounding the *divider* determine the spacing above and below it. For example, headings of differing importance require different spacing above them, but it is possible to apply spacing preselects for space above and/or below the *divider*.
+
+### Usage guidance
+Use *divider* to separate sections that may not otherwise have a clear beginning or end, and to separate information blocks where the context or continuity changes between the blocks.
+
+### Style guidance
+When white space or headings can clearly define new sections, dividers are not needed. They are also not for graphic decoration.
+
+## Accessibility
```

Comment:
I feel like the Accessibility stuff is really all that makes sense in the base here.